lib/rucio/daemons/hermes/hermes.py

```diff
@@ -240,6 +240,45 @@ def aggregate_to_influx(
     return 204
 
 
+def build_message_dict(
+        bulk: int,
+        worker_number: int,
+        total_workers: int,
```

Comment:
Just as an opinion, I think these two could be named the same way they're named in `retrieve_messages` (i.e. `thread` and `total_threads`) for consistency